privoxy/scripts/start.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Source our persisted env variables from container startup
+. /etc/transmission/environment-variables.sh
+
+find_proxy_conf()
+{
+    if [[ -f /usr/local/etc/privoxy/config ]]; then
+      PROXY_CONF='/usr/local/etc/privoxy/config'
+    elif [[ -f /usr/local/etc/privoxy/privoxy/config ]]; then
+      PROXY_CONF='/usr/local/etc/privoxy/privoxy/config'
+    else
+     echo "ERROR: Could not find privoxy config file. Exiting..."
+     exit 1
+    fi
+}
```

Comment:
The reason why we had to check two places was that tinyproxy had different default location for config based on distro. And we used to have both ubuntu and alpine support so we had to figure out where it was. Should not be an issue anymore.

Reply:
So alpine does have a privoxy build, but I chose to build manually because a) more up to date version than the repo b) Being able to compile in and try out the HTTPS MITM support which will greatly enhance Privoxy's effectiveness and ability to change requests on the now mostly HTTPS web (it's still experimental though, so have left it off for now, but a user could enable it simply by changing the config without recompiling)